src/components/Tabs/README.md

```diff
@@ -1,57 +1,314 @@
+    </UIKit.Tabs>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Tabs activeTab="first">
+  <Tabs.Item id="first" title="First Tab" onClick={() => {}} counter={13} />
+  <Tabs.Item id="second" title="Second Tab" onClick={() => {}} counter={3} />
+</Tabs>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Tabs.Item properties
+
+| Name     | Description                      |            Type             | Default |
+| :------- | -------------------------------- | :-------------------------: | :-----: |
+| id       | Tab ID                           |          `String`           |         |
+| title    | Tab title                        | `String`, `React.ReactNode` |         |
+| meta     | Tab description                  |          `String`           |         |
+| hint     | HTML title attribute             |          `String`           |         |
+| icon     | Icon displayed at the left       |      `React.ReactNode`      |         |
+| counter  | Number displayed at the right    |      `React.ReactNode`      |         |
+| label    | `<Label>` displayed at the right |      `React.ReactNode`      |         |
+| disabled | Inactive state                   |          `Boolean`          |         |
+
+## Properties
+
+| Name             | Description                                                          |                                    Type                                    |   Default    |
+| :--------------- | :------------------------------------------------------------------- | :------------------------------------------------------------------------: | :----------: |
+| direction        | Tab direction (deprecated)                                           |                                  `String`                                  | `horizontal` |
+| activeTab        | Active tab ID                                                        |                                  `String`                                  |              |
+| allowNotSelected | Allows `activeTab` to be undefined                                   |                                 `Boolean`                                  |              |
+| items            | Tabs array                                                           |                             `TabsItemProps[]`                              |     `[]`     |
+| onSelectTab      | Select tab handler                                                   |                    `onSelectTab?(tabId: string): void`                     |              |
+| wrapTo           | Allows to wrap `TabItem` into another component or render custom tab | `wrapTo?(item: TabsItemProps, node: React.ReactNode, index: number): void` |              |
+| className        | CSS-class of element                                                 |                                  `String`                                  |              |
```

Comment:
Let's use lowercase for all type primitives

Reply:
done